packages/split-button/src/SplitButton.ts

```diff
@@ -115,6 +115,14 @@ export class SplitButton extends SizedMixin(PickerBase) {
                 this.selects = 'single';
             }
         }
+        if (window.__swc.DEBUG) {
+            window.__swc.warn(
```

Comment:
I wonder if we should introduce a deprecated notice function here, not because it would do something different but we could do different things in the future and have all deprecations do the same thing.

Reply:
FYI, this displays as follows for a development time user: 

<img width="857" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/fa4e5223-0977-4a8d-955c-f8f1f9be1246">